app/components/product/edit.hbs

```diff
@@ -76,6 +76,14 @@
                 class="mt-1 focus:ring-red-200 focus:border-red-200 block w-full border-gray-300 rounded-md sm:text-sm" />
              </div>
           </div>
+          <div class="mt-4 text-xs text-gray-400">
+            <div>
+              <Util::CreatedOn @date={{@model.created}} @user={{@model.creator}} />
+            </div>
+            <div>
+              <Util::UpdatedOn @date={{@model.modified}} @user={{@model.editor}} />
+            </div>
+          </div>
```

Comment:
Zie je een specifieke meerwaarde om die provenance attributes ook in de edit-mode te tonen? Ik ben geneigd ze enkel in view-mode te tonen.

Reply:
Het hangt af van de oorspronkelijke motivatie voor het tonen van de modified-dates, maar als de modified date/editor de beslissing om al dan niet nog eens te editen kan beïnvloeden, lijkt het me handig om ze ook in edit mode nog te zien staan ...